ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml

```diff
@@ -1689,11 +1689,17 @@ presubmits:
         - ci-operator
         env:
         - name: CLUSTER_TYPE
-          value: openstack
+          value: openstack-vexxhost
```

Comment:
Shouldn't we modify ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml instead?

Reply:
Would it be possible to switch by using ‘OS_CLOUD’ below?

Reply:
Sorry, I meant this change should probably go in `ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml` rather than `ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml`.